src/Illuminate/Database/Schema/Grammars/PostgresGrammar.php

```diff
@@ -22,6 +23,22 @@ class PostgresGrammar extends Grammar
     protected $serials = ['bigInteger', 'integer', 'mediumInteger', 'smallInteger', 'tinyInteger'];
 
     /**
+     * Get the bindings for determining table information including schema name.
+     *
+     * @param Connection $connection
+     * @param            $table
```

Comment:
param annotations must include type data otherwise the syntax is actually invalid